### ISSUE TITLE
The secureData.getWebOwners will make the value from capital to small letter

### ISSUE THREAD

**Issue by haoxli** [01org/AppSecurityApi#3]
Set webOwners as 'webOwners1' in createFromData(), but get 'webowners1' in getWebOwners().

```
intel.security.secureData.createFromData(
        function(instanceID) {
          intel.security.secureData.getWebOwners(
            function(webOwners){
              t.step(function() {
                assert_equals(webOwners, 'webOwners1');
              });
              t.done();
            },
            function(errorObj) {
              t.step(function() {
                assert_true(false, 'fail:code = ' + errorObj.code + ', message = ' + errorObj.message);
              });
              t.done();
            },
            instanceID
          );
        },
        function(errorObj) {
          t.step(function() {
            assert_true(false, 'fail: code = ' + errorObj.code + ', message = ' + errorObj.message);
          });
          t.done();
        },
        {'data': 'plaintext data', 'tag': 'Test', 'extraKey': 0, 'appAccessControl': 0, 'deviceLocality': 0, 'sensitivityLevel': 0, 'noStore': false, 'noRead': false, 'creator': 0, 'owners': [0], 'webOwners': 'webOwners1'}
      );
```

**Comment by OhadBoimelIntel** [comment]
Hi @haoxli ,
Thank you for the report, we will take a look and check it.

**Comment by OhadBoimelIntel** [comment]
We acknowledge this issue, it will be fixed in future version

**Comment by OhadBoimelIntel** [comment]
Fixed in version 1.4.0
